optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -776,6 +787,10 @@ class LightGBMTuner(_LightGBMBaseTuner):
                 Progress bars will be fragmented by logging messages of LightGBM and Optuna.
                 Please suppress such messages to show the progress bars properly.
 
+        optuna_seed:
+            A parameter specifies random seed of TPESampler used in tuning num_leaves,
+            bagging_fraction, bagging_freq, lambda_l1 and lambda_l2.
```

Comment:
```suggestion
            ``seed`` of :class:`~optuna.samplers.TPESampler` for random number generator.
```

Reply:
Thank you for your comment. I basically agree with your proposal but I think it might be better to specify which parameters a tuned by TPESampler since min_child_samples and feature_fraction are tuned by other algorithm. How do you think?

Reply:
Indeed! Thank you for your clarification; their samplers are `GridSearch`. I have updated my two suggestions.